### DISCUSSION TITLE
PR merged without pass checked github actions

### DISCUSSION THREAD

**Discussion by zzau13** [community/community#48345] [Pull Requests]
### Select Topic Area

Bug

### Body

https://github.com/node-cache-manager/node-cache-manager/pull/364 checks fails but PR is automerge. Thanks!

**Comment by jge162** [comment]
You sure the branch protection settings are correct? Seems based on that pull request they are not.

**Comment by zzau13** [comment]
![image](https://user-images.githubusercontent.com/2625545/220906813-b4a81818-33ca-49a0-b20b-460f1b708fab.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @botika ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
